test/test.util.js

```diff
@@ -71,10 +77,18 @@ function testListIsPunycode (t, list) {
 }
 
 function testListDoesntContainRepeats (t, list) {
-  list.forEach((domain) => {
-    const count = list.filter(item => item === domain).length
-    t.ok(count === 1, `domain '${domain}' is duplicated. Domains can only appear in list once`)
-  })
+  const clone = new Set(list);
+  if (clone.size === list.length) {
+    return;
+  }
+  
+  for (const item of list) {
+    if (clone.has(item)) {
+      clone.delete(item);
+    } else {
```

Comment:
Nit: This might be a simpler way of accomplishing the same thing:
```suggestion
    if (!clone.has(item)) {
```

Reply:
I don't think that works, clone will always have item, but list might have the same item multiple times while clone only has it once

Reply:
Oh I see, I misread what was happening here